doc/MBU_installation_manual.md

```diff
@@ -0,0 +1,180 @@
+# MBU Installation and User Manual
+
+## 1 Introduction
+
+### 1.1 Background/Context
+
+The NetCDF to BUFR _conversion_ tool MBU processor was initially developed by SERCO for ESA. 
+
+CLS took over the responsibility to maintain this software. Starting from version 2.0, the maintenance is thus performed by CLS. 
+
+### 1.2 Purpose of this document
+
+The purpose of this document is to describe:
+-	The objective, architecture, and way to use the NetCDF to BUFR conversion tool.
+-	The requirements before installation 
+-	The procedure for installation
+
+### 1.3 Document organisation
+
+Section 1: this introduction
+
+Section 2: the installation manual
+
+Section 3: the user manual
+
+Section 4: the development manual
+
+### 1.4 Applicable and Reference Documents
+
+#### Applicable Documents
+
+| [IPF-ICD] | ESA Generic Processor ICD | GMES-GSEG-EOPG-TN-090016 Issue 1, Revision 0, 24 sept 2009 |
+|-----------|:-------------------------:|-----------------------------------------------------------:|
+
+#### Reference Documents
+
+The following documents provide useful reference information associated with this document.  These documents are to be used for information only.  Changes to the date/revision number (if provided) do not make this document out of date.
+
+| [BUFR-TR] | NetCDF to BUFR 3.0 Validation Report | DI-MPC-BUFR-VR MPC-0500 version 1.0 |
+|-----------|--------------------------------------|-------------------------------------|
+
+### 1.5 Acronyms and Definition
+
+**IPF** &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Instrument Processing Facility  
+**MBU** &nbsp;&nbsp;&nbsp;   	Meteo BUfr : processor for conversion of S1 WV OCN products into Bufr format
```

Comment:
I would have preferred to avoid this kind of explicit markup with "&nbsp;", but indeed there seems to be no better way to do align the text